src/components/List/List.stories.mdx

```diff
@@ -168,6 +181,22 @@ export const definitions = [
 
 <ArgsTable story="List with Controls" />
 
+## Accessibility
+
+Internally, the `List` component implements native HTML markup for unordered,
+ordered, and definition lists. This includes `ul`, `ol`, and `dl` elements.
```

Comment:
Just an FYI: I hadn't heard of "definition lists" before and looked it up. MDN calls them "description lists" (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl). Though I did see a couple of instances of other sites calling them "definition lists".

Reply:
That's interesting. It seems to be interchangeable including on w3c pages. [This](https://www.w3.org/WAI/WCAG21/Techniques/html/H40.html) is a good example that says that "Description list" was introduced in HTML5 but before it was called "Definition list". I have yet to encounter "Description list".

Let's make this update but it would require a much larger change to enums and other docs. Some resources: 

- https://www.w3.org/WAI/WCAG21/Techniques/html/H40.html
- https://html.spec.whatwg.org/multipage/grouping-content.html#the-dl-element

@bigfishdesign13 what do you think?

Reply:
Wow, I didn’t realize this either.  That said, I would like to keep aligned with the latest naming conventions, so yes, we should make this change.  Unfortunately, that means before `v1.0`, as a change like this, which would remove “ListTypes.Definition” from the enum values, would be a breaking change.